maestro-test/src/test/resources/020_parse_config.yaml

```diff
@@ -1,9 +1,5 @@
-- config:
```

Comment:
The purpose of having a `Map` and this test in particular is to be able to parse-out mobiledev-specific fields without having open-source Maestro knowing about concrete types.

Reply:
Updated